x-pack/plugin/security/src/test/java/org/elasticsearch/test/SecuritySettingsSource.java

```diff
@@ -90,23 +101,11 @@
         + TEST_USER_NAME
         + ","
         + DEFAULT_TRANSPORT_CLIENT_USER_NAME
-        + "\n"
-        + DEFAULT_TRANSPORT_CLIENT_ROLE
-        + ":"
-        + DEFAULT_TRANSPORT_CLIENT_USER_NAME
-        + "\n"
-        + "superuser:"
-        + TEST_SUPERUSER
-        + "\n";
+        + "\n" //
+        + (DEFAULT_TRANSPORT_CLIENT_ROLE + ":" + DEFAULT_TRANSPORT_CLIENT_USER_NAME + "\n") //
```

Comment:
Nit: unused trailing comments

Reply:
They're not unused. They're forcing spotless to not reformat 😈

Reply:
TIL!